LondonBritishLibrary/orient/BLorient793.xml

```diff
@@ -26,14 +25,781 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+</msItem>
+
+<msItem xml:id="ms_i1.53">
+<locus from="12v"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter53">Of Pentecost</title>
+<incipit xml:lang="gez"><locus target="#4r"></locus>በእንተ፡ ጰንተቈስጤ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.54">
+<locus from="13r"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter54">Of those who are invited with the bishop (to consume the sacred elements)</title>
+<incipit xml:lang="gez"><locus target="#13r"></locus>በእንተ፡ እለ፡ ምስለ፡ ኤጲስ፡ ቆጶስ፡ ተፀውዑ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.55">
+<locus from="13r"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter55">Of fruits which are offered</title>
+<incipit xml:lang="gez"><locus target="#13r"></locus>በእንተ፡ ፍሬ፡ ዘያበውዑ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.56">
+<locus from="13r"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter56">Of the order of a wedding</title>
+<incipit xml:lang="gez"><locus target="#13r"></locus>በእንተ፡ ጾታ፡ መርዓት፡ ዘቤተ፡ ክርስቲያን።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.57">
+<locus from="13r"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter57">That the Holy Eucharist is to be sent to a neophyte who is sick</title>
+<incipit xml:lang="gez"><locus target="#13r"></locus>በእንተ፡ ከመ፡ ይሰዱ፡ አኩቴተ፡ ለንዑሰ፡ ክርስቲያን፡ ድውይም።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.58">
+<locus from="13v"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter58">Of those who die in poverty</title>
+<incipit xml:lang="gez"><locus target="#13v"></locus>በእንተ፡ ለእመቦ፡ ዘሞተ፡ ነዳይ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.59">
+<locus from="13v"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter59">Of the prayers of the people</title>
+<incipit xml:lang="gez"><locus target="#13v"></locus>በእንተ፡ ዘከመ፡ ይጸልዩ፡ ሕዝብ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.60">
+<locus from="13v"/>
+<title type="complete" xml:lang="en" ref="LIT2461Testam#Chapter60">Of the receiving of the Holy Eucharist by the faithful</title>
+<incipit xml:lang="gez"><locus target="#13v"></locus>በእንተ፡ ዘከመ፡ ይትመጠው፡ አኰቴተ፡ ምእመናን።</incipit>
+<explicit xml:lang="gez"><locus target="#13v"/>ወሶቤሃ፡ ወደቅነ፡ ወሰገድነ፡ ሎቱ፡ እንዘ፡ ንብል፡ ስብሐት፡ ለከ፡ ኢየሱስ፡ ዘብርሃን፡ ስሙ፡ ዘወሀብከነ፡ ዘእንተ፡ ታፈቅር፡ ከመ፡ ንኩን፡ ከማከ፡ ንሕነ፡ ወኵሎ፡ ዘተሰምዓከ፡
+ወተናጊሮ፡ ምስሌነ፡ ዘንተ፡ ኵሎ፡ ወወሀበነ፡ ሥልጣነ፡ ፍትሕ፡ ወኃይል፡ ብዙኅ፡ ወዓርገ፡ እምኔነ፡ ሰላሞ፡ ኃዲጎ፡ ለነ። ዮሐንስ፡ ወጴጥሮስ፡ ወማቴዎስ፡ ጸሐፍዎ፡ ለዝ፡ ኪዳን፡ እንዘ፡ ያጽሕፎሙ፡ እግዚአብሔር፡ እግዚእነ፡
+ወፈነዉ፡ በአርአያዝ፡ ኪዳን፡ ግፅወ፡ እምኢየሩሳሌም፡ ደሲቴዎስ፡ ወሲላስ፡ ወመግኖስ፡ ወአቂላስ፡ እለ፡ ኃረዩ፡ ለመላእክት፡ ኵሎ፡ በሓውርተ።</explicit>
+</msItem>
+
+<msItem xml:id="ms_i1.61">
+<locus from="14r"/>
+<title type="complete" xml:lang="en" ref="LIT2462Testam">Discourse of our Lord to his disciples in Galilee, after his resurrection</title>
+<note>The text is partially transcribed in <ref target="DiscourseInGalilee"/>.</note>
+</msItem>
+
+<msItem xml:id="ms_i1.61.1">
+<locus from="14r"/>
+<title type="supplied">Beginning</title>
```

Comment:
the value supplied does not apply to the title. We use either complete or incomplete. What do you mean by supplied?